Команда в консоли:
PS C:\Akademia_Avito\Питон\HW_zanyatie_9\issue-03> python -m unittest -v test_fit_transform_unit.py


Результат:
test_exception (test_fit_transform_unit.TestFitTransform) ... ok
test_one_two_three (test_fit_transform_unit.TestFitTransform) ... ok
test_qwerty (test_fit_transform_unit.TestFitTransform) ... ok
test_seqw (test_fit_transform_unit.TestFitTransform) ... ok
test_two (test_fit_transform_unit.TestFitTransform) ... ok

----------------------------------------------------------------------
Ran 5 tests in 0.002s

OK
